lib/Doctrine/DBAL/Schema/Visitor/CreateSchemaSqlCollector.php

```diff
@@ -47,15 +48,15 @@ public function acceptNamespace($namespaceName)
     /**
      * {@inheritdoc}
      */
-    public function acceptTable(Table $table)
+    public function acceptTable(Table $table) : void
     {
-        $this->createTableQueries = array_merge($this->createTableQueries, $this->platform->getCreateTableSQL($table));
+        $this->createTableQueries = array_merge($this->createTableQueries, array_values($this->platform->getCreateTableSQL($table)));
```

Comment:
@Ocramius my major concern about adding `array<int, T>` is the need for such `array_values()` calls here and there. Specifically, in this case, it seems to massage the `string[]` returned by `AbstractPlatform::getCreateTableSQL()` to `array<int, string>` (the type of `$this->createTableQueries`) for no real reason.